helix-core/src/test/java/org/apache/helix/monitoring/TestClusterStatusMonitorLifecycle.java

```diff
@@ -164,46 +136,41 @@ public void beforeClass() throws Exception {
   @AfterClass
   public void afterClass() throws Exception {
     System.out.println("Cleaning up...");
-    for (int i = 0; i < 2 * n; i++) {
-      if (_controllers[i] != null) {
-        _controllers[i].syncStop();
-      }
-    }
     for (MockParticipantManager participant : _participants) {
       if (participant != null) {
         participant.syncStop();
       }
     }
-    cleanupControllers();
```

Comment:
Umm why are we removing this? We still want to remove all controllers at the end of this test case right?